### DISCUSSION TITLE
WebSocketSubjectConfig documentation page is not available.

### DISCUSSION THREAD

**Discussion by d-sainz** [ReactiveX/rxjs#7037] [Report issues other than bug]
While reading the docs for the `webSocket` function I tried to go to the documentation for the `WebSocketSubjectConfig` through the link listed in the page, but I got a page not found error.

`webSocket` function documentation: https://rxjs.dev/api/webSocket/webSocket
![imagen](https://user-images.githubusercontent.com/74557072/184127673-d09d075e-4968-4698-9ac7-be44c15c0df3.png)

`WebSocketSubjectConfig` link: https://rxjs.dev/WebSocketSubjectConfig
![imagen](https://user-images.githubusercontent.com/74557072/184127916-7c3d3f7e-572a-4852-8dac-bd815a44f1b3.png)

Evenually I found unofficial documentation for the object, but I think it should be up with the rest of the rxjs docs.

**Comment by jakovljevic-mladen** [comment]
This is fixed in https://github.com/ReactiveX/rxjs/pull/6943 and it's now published to [the official docs](https://rxjs.dev/api/webSocket/WebSocketSubjectConfig). Thanks for reporting.
